src/NServiceBus.Core.Tests/ApprovalFiles/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1492,36 +1504,43 @@ namespace NServiceBus.MessageMutator
 }
 namespace NServiceBus.ObjectBuilder.Common
 {
-    public interface IContainer : System.IDisposable
```

Comment:
Why does this one only obsolete the interface and not the members while `IBuilder` does the interface as well as the members (I think the later is the more correct way of obsoleting)

Reply:
because IContainer shouldn't be used directly by the user while IBuilder is a very common user-facing API. IContainer is only used by downstream packages to implement the DI seam.